src/datasources/data-frame/DataFrameDataSource.ts

```diff
@@ -152,7 +151,12 @@ export class DataFrameDataSource extends DataSourceBase<DataFrameQuery> {
     }
   }
 
-  private convertDataFrameFields(frame: DataFrame, columns: Column[]): DataFrame {
+  private convertDataFrameFields(tableData: TableData, columns: Column[]): DataFrame {
+    this.transformBooleanFields(tableData, columns);
+    const frame = toDataFrame(tableData);
+    frame.fields.forEach(field => {
+      field.config.displayName = field.name;
+    })
```

Comment:
I think we should only specify this when the column's name is 'value', otherwise it could technically be a breaking change. Some panels such as the graph let you set a display name, but it doesn't work when the field's configuration is set programmatically like this. The workaround is pretty simple and I'm not sure anyone will care, but my preference would be to scope the change smaller.

![image](https://github.com/ni/systemlink-grafana-plugins/assets/9257800/b69fe523-a36f-4968-b17d-39d49db6ccc3)